### PR TITLE
#446 Two Step Verification Automation

### PR DESCRIPTION
**Pull request by lexotrion**
I extracted console two step verification into separate class that implements new TwoStepVerificationInterface. So everyone can make their own two step verification automation class.

As you can see, i made an example of verification automation via email in `instagram-php-scraper/examples/twoStepAutoVerification/loginWithEmailAutoCponfirm.php` -

### CODE REVIEW

**Comment by lexotrion** [comment]
[raiym] any feedback?

**Comment by raiym** [comment]
@lexotrion looks great. 
need more time for evaluation.

File: src/InstagramScraper/sessions/cli/Files/.htaccess [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<IfModule mod_authz_host>
```
Comment (raiym) [inline src/InstagramScraper/sessions/cli/Files/.htaccess]: Do we need to keep this file?

Comment (lexotrion) [reply src/InstagramScraper/sessions/cli/Files/.htaccess]: I think not. I'll drop it

**Comment by raiym** [comment]
@lexotrion  thank you for contribution.

**Pull request merged**